### PR TITLE
Add missing useEffect dependencies - Radio Schedules

### PR DESCRIPTION
**Pull request by sareh**
No ticket. Just a fix for this warning when running tests:

```
121:6  warning  React Hook useEffect has missing dependencies: 'timeOnClient' and 'timeOnServer'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
11:22:32 [Test Development] 
11:22:32 [Test Development] ✖ 1 problem (0 errors, 1 warning)
```
**Overall change:** 
Ensure useEffect has all required dependencies

**Code changes:**
- Ensure timeOnClient & timeOnServer are in useEffect dependencies list

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by chris-hinds — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Comment by PriyaKR** [comment]
This doesn't require any testing from a tester.

File: src/app/containers/RadioSchedule/Canonical/index.jsx [hunk 1]
```diff
@@ -128,7 +128,7 @@ const CanonicalRadioSchedule = ({ endpoint }) => {
         .catch(e => logger.error(`HTTP Error: "${e}"`));
 
     fetchRadioScheduleData(endpoint);
-  }, [endpoint, service, script, timezone, locale]);
+  }, [endpoint, locale, script, service, timeOnClient, timeOnServer, timezone]);
```
Comment (tochwill) [inline src/app/containers/RadioSchedule/Canonical/index.jsx]: I think this is going to lead to an infinite loop where we keep requesting data from the client - `timeOnClient` is calculated outside of the useEffect hook. The value of this is going to be different on every render, which means the useEffect is going to be called continuously. I just tried adding this to the list of deps and saw this happen in the server logs. We can move the calculation of `timeOnClient` inside the hook and remove it from the deps to fix this :)

**Review by tochwill — CHANGES_REQUESTED** [review]

**Comment by sareh** [comment]
After moving the timeOnClient, I checked the page 
http://localhost:7080/hausa?renderer_env=live
hausa front page with radio schedule
and storybook: 
http://localhost:9001/iframe.html?id=containers-radioschedule--radio-schedules&knob-Select%20a%20service=hausa
Hausa storybook
and they're all still working well. In fact, the knobs on storybook are now working locally, due to this being fixed.

File: src/app/containers/RadioSchedule/Canonical/index.jsx (hunk 1 above)
Comment (sareh) [reply src/app/containers/RadioSchedule/Canonical/index.jsx]: @tochwill Thanks. This has been fixed now.

**Review by tochwill — APPROVED** [review]

**Pull request merged**